wandb/util.py

```diff
@@ -1552,6 +1553,34 @@ def load_as_json_file_or_load_dict_as_json(config: str) -> Any:
             return None
 
 
+def _parse_entity_project_item(path: str) -> tuple:
```

Comment:
`_parse_artifact_path` pulls entity and project from the settings if they don't exist but that wouldn't work for our use case, where the target portfolio might be under a different entity/project combination. I _could_make it work by modifying this function...but in any case, I don't like the idea of resolving entity and project inside the parse function. The parse function should only be focused on parsing the given path and extracting what it can, returning None or "" for the parts it can't find within the path. Which is why I created `_parse_entity_project_item`.